packages/api/tests/lambdas/test-sns-consumer.js

```diff
@@ -111,16 +84,10 @@ test.serial('it should enqueue a message for each SNS rule', async (t) => {
     workflow: 'test-workflow-1',
   };
 
-  const createdRule = await ruleModel.create(rule1);
-  const expectedRule = {
-    ...createdRule,
-    meta: {
-      snsSourceArn: snsArn,
-    },
-  };
+  fetchEnabledRulesStub.returns(Promise.resolve([rule1]));
   await handler(event, {}, testCallback);
 
   t.is(queueMessageStub.callCount, 1);
-  t.deepEqual(queueMessageStub.getCall(0).args[0], expectedRule);
+  t.deepEqual(queueMessageStub.getCall(0).args[0], rule1);
```

Comment:
Eyeballing the diff here, it seems like previously the test would have expected `rule1.meta.snsSourceArn` but does not anymore? Is that logic change intentional?

Reply:
Good catch, I'll fix the expectation.

Reply:
https://github.com/nasa/cumulus/pull/2296/commits/58441b880cb6f2084dac63113a0dddda10553183